src/com/clover/remote/client/CloverConnector.ts

```diff
@@ -145,7 +145,7 @@ export class CloverConnector implements sdk.remotepay.ICloverConnector {
      * @param request
      * @param suppressTipScreen
      */
-    private saleAuth(request: sdk.remotepay.TransactionRequest, suppressTipScreen: boolean): void {
```

Comment:
suppressTipScreen was not being used.  It looks like it used to be but when tipMode was introduced it's usages were also removed.